tests/features/bootstrap/FeatureContext.php

```diff
@@ -1038,4 +1039,52 @@ public function assertPageNotCached() {
     $this->assertSession()->responseHeaderContains('X-Drupal-Cache', 'MISS');
   }
 
+  /**
+   * Clears the static cache of DatabaseCacheTagsChecksum.
+   *
+   * Static caches are typically cleared at the end of the request since a
+   * typical web request is short lived and the process disappears when the page
+   * is delivered. But if a Behat test is using DrupalContext then Drupal will
+   * be bootstrapped early on (in the BeforeSuiteScope step). This starts a
+   * request which is not short lived, but can live for several minutes while
+   * the tests run. During the lifetime of this request there will be steps
+   * executed that do requests of their own, changing the state of the Drupal
+   * site. This does not however update any of the statically cached data of the
+   * parent request, so this is totally unaware of the changes. This causes
+   * unexpected behaviour like the failure to invalidate some caches because
+   * DatabaseCacheTagsChecksum::invalidateTags() keeps a local storage of which
+   * cache tags were invalidated, and this is not reset in time.
+   *
+   * For this reason, in such limited cases, where we need to clear the cache
+   * tags cache, we tag the Behat feature with @clearStaticCache. This ensures
+   * that static cache is cleared after each step.
+   *
+   * CAUTION: Use the @clearStaticCache tag only in scenarios where you have
+   * trouble with the static caching being preserved across step requests,
+   * because clearing the static cache too often might affect performance.
+   *
+   * @see \Drupal\Core\Cache\DatabaseCacheTagsChecksum
+   * @see https://github.com/jhedstrom/drupalextension/issues/133
+   *
+   * @AfterStep
+   */
+  public function clearCacheTagsStaticCache(AfterStepScope $event) {
+    $feature = $event->getFeature();
+    if ($feature->hasTag('clearStaticCache')) {
+      parent::clearStaticCaches();
```

Comment:
I do not think that this is an appropriate approach. I checked the implementation a bit and this is what I found:
If we keep this one, the code for cache within the LastUpdateFormatter class which creates the og-group-content cache tag is not needed.
If we keep the code for caching within the LastUpdateFormatter class, then this static clear is not needed.

Searching a bit more I noticed that in `og_invalidate_group_content_cache_tags` the last line has the `Cache::invalidateTags(Cache::buildTags('og-group-content', $tags));` which is meant to invalidate the cache related to the group and the group-content. 
The above seems to work fine but is not true when it comes to computed fields. The problem is that og does not invalidate the parent tags rather than simply their own built tags.
For our computed field, we need to have the value changed on load. Instead, when the group is first loaded, the field gets statically cached with the first value. The og cache invalidation correctly invalidates all content including the parent but not the render elements which are already in cache with the previous value. 

I think the proper way is for og to invalidate the group cache tags instead of only the og-group-context: prefixed ones.
What do you think?

Reply:
Disregard the part that this piece of code works without the corresponding code from the LastUpdateFormatter. I could reproduce the issue and this is actually needed. Maybe we should have a ticket related to check more on that subject.